src/color/setting.js

```diff
@@ -773,14 +773,31 @@ function setting(p5, fn){
    *
    * `OKLCH`  - An easier-to-use representation of OKLAB, expressing colors in terms of Lightness, Chroma, and Hue.
    *          This mode retains the perceptual benefits of OKLAB while offering a more intuitive format for color manipulation.
-   * 
+   *
    * <a href="#/p5.Color">p5.Color</a> objects remember the mode that they were
    * created in. Changing modes doesn't affect their appearance.
    *
+   *  `Single-value (Grayscale) Colors`:    
+   *  When a color is specified with only one parameter (e.g., `color(g)`), p5.js will interpret it
+   *  as a grayscale color. However, how that single parameter translates into a grayscale value
+   *  depends on the color mode:
+   *
+   * - `RGB, HSB, and HSL`: The single value will be used consistently for the gray equivalent 
+   *    in each of these modes (for example, mapping to red/green/blue channels equally in RGB).
```

Comment:
RGB uses blue max (not sure if it is worth to change it).

HSB and HSL uses Brightness and Lightness max respectively with hue and saturation at 0, at this mode, both color spaces are equivalent.